docs/moz-addon-manager.md

```diff
@@ -30,6 +30,10 @@ Go to `about:config` in Firefox and set this property to `true`:
 xpinstall.signatures.dev-root
 ```
 
+If you do not see an entry for `xpinstall.signatures.dev-root` in `about:config` you will need to add one by doing the following: 
+
+Right click in `about:config` in Firefox , select `new` and then add `xpinstall.signatures.dev-root` as `Boolean`.
```

Comment:
I think you can remove the `in Firefox` here, as we've already said above "you will need to tell Firefox to honor their signing certificates".